File: db4on/reference/db4o-reference-chapters/src/com/db4odoc/f1/lists/CollectionFactory.cs
/* Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com */
using System;
using System.Collections;

namespace com.db4odoc.f1.lists
{
	public class CollectionFactory
	{
		public static IList newList()
		{
			return  new VerboseList();
		}
	}
}

File: db4on/reference/vb/db4o-reference-chapters/src/com/db4odoc/f1/lists/CollectionFactory.vb
' Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com 

Imports System
Imports System.Collections

Namespace com.db4odoc.f1.lists
    Public Class CollectionFactory
        Public Shared Function NewList() As IList
            Return New VerboseList()
        End Function
    End Class
End Namespace

